Main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Unit.h"
+#include "Control.h"
+
+int main(int argc,char** argv)
+{
+	Control vezerles;

```

Comment:
Igazabol ha static-ok lennenek a metodusok, akkor nem is kell peldanyositani, ugysincs benne semmi state.